Fridge/Manager.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Fridge
+{
+    public class Manager
+    {
+        Fridge myFridge = new Fridge();
```

Comment:
Field should be initialized in a constructor.
Missed access modifier for field.

Reply:
-1 to source code quality.